portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -90,4 +105,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/comments.html");
   }
+
+  /* Returns the value of maximum number of comments requested, or -1 if the input is invalid. */
```

Comment:
Should this comment utilize the JavaDoc tags as well?

Reply:
Yes the Javadoc was added in PR #15.